locales/en/public.json

```diff
@@ -141,23 +141,23 @@
     },
     "TITLE": "{{chartKind}} (last {{duration}}s, every {{period}}s)"
   },
-  "ClickableAutomatedAnalysisLabel": {
+  "ClickableAutomatedAnalysisresult": {
     "ARIA_LABELS": {
-      "POPOVER": "automated-analysis-description-popover"
+      "POPOVER": "cryostat_tmp"
```

Comment:
What's happening here?

Reply:
Here too! I think this is `yarn localize`.

Reply:
https://github.com/cryostatio/cryostat-web/blob/main/LOCALIZATION.md

> The value of the key will be the string cryostat_tmp by default (we set this in i18next-parser.config.js). This is a placeholder value that should be replaced with the actual translation by going to the corresponding locale json file and manually replacing the value with the translation.

I guess this is because, for whatever reason, this string is included in the translation table but doesn't have any actual translations. It seems like this string is not actually used for graphical display in the UI but for assigning an accessibility label ("aria") to a popover element. This also seems like something that should not be changed in this PR.